packages/db/test/user.spec.js

```diff
@@ -183,23 +183,6 @@ describe('user operations', () => {
       dagSize: dagSize2
     })
 
-    // Create "Failed" Upload. It should not be counted.
-    const cid3 = await randomCid()
-    const dagSize3 = 100000
-    await createUpload(client, user._id, authKey._id, cid3, {
-      dagSize: dagSize3,
-      pins: pinsError
-    })
-
-    // Create Upload not pinned yet. It should not be counted yet.
-    await createUpload(client, user._id, authKey._id, await randomCid(), {
-      dagSize: 1000000,
-      pins: initialPinsNotPinned
-    })
-
-    const usedStorageWithFailed = await client.getStorageUsed(user._id)
-    assert.strictEqual(usedStorageWithFailed.uploaded, dagSize1 + dagSize2, 'used storage should not count unpinned')
-
```

Comment:
These would be accounted now, so really no need to add them given there is no exception anymore